libdevcore/Common.h

```diff
@@ -294,6 +294,12 @@ enum class WithExisting: int
 	Kill
 };
 
+/// Get the current time in seconds since the epoch in UTC
+inline time_t getNowUTC()
```

Comment:
By moving it to cpp you will hide the implementation details. There is also no benefits from inlining as you need to do 2 lib calls. The users will not need to include additional headers as <ctime>.

The result is usually converted to u256, but not always. @LefterisJP replaced one case where it is used as a RNG seed. And if I remember correctly, uint64_t is implicitly convertible to u256.